cfme/rest/__init__.py

```diff
@@ -354,3 +355,23 @@ def mark_vm_as_template(rest_api, provider, vm_name):
         lambda: rest_api.collections.templates.find_by(name=vm_name).subcount != 0,
         num_sec=700, delay=15)
     return rest_api.collections.templates.get(name=vm_name)
+
+
+def arbitration_settings(request, rest_api, num=2):
```

Comment:
@psav This file is collection of helper methods for REST API collections. With few exceptions all functions in this file are creating random data. My understanding of the purpose of these functions is that authors of new tests don't have to figure out how the request for particular collection should look like.